packages/app-admin-users-cognito/src/plugins/userMenu/accountDetails.tsx

```diff
@@ -41,7 +47,7 @@ export const AccountDetails = () => {
     }
 
     return (
-        <Link to={"/account"} className={linkStyles}>
+        <Link to={accountRoute} className={linkStyles}>
```

Comment:
Didn't realize we don't show Account details dropdown menu item if not on default tenant. 💡